services/PostService.js

```diff
@@ -1,10 +1,17 @@
 const Post = require("../models/index").posts;
+const Comment = require("../models/index").comments;
 const { Op } = require("sequelize");
 
 module.exports = class PostService {
   static async findAll() {
     try {
-      const allPost = await Post.findAll();
+      const allPost = await Post.findAll({
```

Comment:
So the front end should make a separate API call asking for comments about a specific post rather than the backend making the heavy lifting?

Reply:
Just create a new endpoint and it will be use for comment and post. The finding all post might be needed, so just leave it in the code, don't need it to be change to finding comments and post.